src/docs/implementations/dynatrace.adoc

```diff
@@ -130,45 +130,95 @@ When using the https://www.dynatrace.com/support/help/dynatrace-api/environment-
-    # or https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest on Managed
+    # Required only if not using the OneAgent endpoint
+    # For SaaS: https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
+    # For managed deployments: https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest
+    uri: ${DT_METRICS_INGEST_URL}
 
     # It is *not* recommended to put the token into the YAML or properties file directly.
     # It should be read from a secure source instead.
-    # E.g. using `api-token: ${ENV_VAR_DT_INGEST_TOKEN}` to read from the environment.
-    api-token: MY_TOKEN
+    # E.g. using `api-token: ${DT_METRICS_INGEST_API_TOKEN}` to read from the environment.
+    api-token: ${DT_METRICS_INGEST_API_TOKEN}
 
-    # Sets a prefix that is prepended to each exported metric key.
-    metric-key-prefix: my.metric.key.prefix
+    # These properties can only be used with the v2 exporter.
+    v2:
+        # Sets a prefix that is prepended to each exported metric key.
+        metric-key-prefix: my.metric.key.prefix
 
-    # If set to true and a local OneAgent or operator is running, retrieves metadata
-    # and adds it as additional dimensions to all data points (default: true)
-    enrich-with-dynatrace-metadata: true
+        # If set to true and a local OneAgent or operator is running, retrieves metadata
+        # and adds it as additional dimensions to all data points (default: true)
+        enrich-with-dynatrace-metadata: true
 
-    # Sets an arbitrary number of key-value pairs as default dimensions.
-    # Micrometer tags will overwrite these dimensions, if the same key appears twice.
-    # Each exported metric will contain these dimensions.
-    default-dimensions:
-        key1: "value1"
-        key2: "value2"
+        # Sets an arbitrary number of key-value pairs as default dimensions.
+        # Micrometer tags will overwrite these dimensions, if they have the same key.
+        # Each exported metric will contain these dimensions.
+        default-dimensions:
+            key1: "value1"
+            key2: "value2"
 
     # The export interval in which metrics are sent to Dynatrace (default: 60s).
     step: 60s
 ----
 
-These properties can also be set in code by overwriting the respective methods of the `DynatraceConfig` class.
+These properties can also be set in code by overwriting the respective methods of the `DynatraceConfig` class:
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String uri() {
+        // The endpoint of the Dynatrace Metrics API v2 including path, e.g.:
+        // "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest".
+        // Should be read from a secure source and not hard-coded.
+        return MY_DYNATRACE_URI;
```

Comment:
```suggestion
        return System.getenv("DT_METRICS_INGEST_URL");
```

Reply:
@arminru I have crafted a proposal, PTAL and let me know what you think.